CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## v0.0.3
+
+*September 1, 2020*
+
+This release focuses mostly on the IBC handler framework and implementation of ICS02 and ICS07.
```

Comment:
Maybe we can start getting used to avoiding the word "handler". 
(File `adr-003-handler-implementation.md` should be renamed, btw.)

```suggestion
This release focuses mostly on the IBC message processor framework and implementation of ICS02 and ICS07.
```